lib/network/kmip/transport/TransportTemplate.ts

```diff
@@ -90,9 +90,14 @@ export default class TransportTemplate {
         readyCallback: (error: Error | null) => void,
     ) {
         try {
+            const options = {
+                ...this.options.tls,
+                host: '127.0.0.1',
+                family: 4,         
```

Comment:
is there a constant somewhere, instead of using `4` ?
otherwise, can we at least use the string representation instead ('IPv4') ?